ivy/functional/frontends/torch/tensor.py

```diff
@@ -421,6 +421,9 @@ def tril(self, diagonal=0):
     def index_select(self, dim, index):
         return torch_frontend.index_select(self._ivy_array, dim, index)
 
+    def std(self, dim, unbiased, keepdim=False):
```

Comment:
`unbiased` should be set to `True` by default. See [here](https://pytorch.org/docs/stable/generated/torch.Tensor.std.html#torch.Tensor.std) 😄